microreactors/gcmr/assembly/RIA_steady_state/Griffin_steady_state.i

```diff
@@ -0,0 +1,227 @@
+[Mesh]
+  [fmg]
+    type = FileMeshGenerator
+    file = '../MESH/Griffin_mesh.e'
+  []
+  [fmg_id]
+    type = SubdomainExtraElementIDGenerator
+    input = fmg
+    subdomains = '10 100 102 103 200 201 300 400 401 500 600 8000 8001'
+    extra_element_id_names = 'material_id equivalence_id'
+    extra_element_ids = '803 802 810 806 807 807 804 801 801 808 809 805 805;
+                         803 802 810 806 807 807 804 801 801 808 809 805 805'
+  []
+  [coarse_mesh]
+    type = GeneratedMeshGenerator
+    dim = 3
+    nx = 10
+    ny = 10
+    nz = 20
+    xmin = -0.1
+    xmax = 0.1
+    ymin = -0.1
+    ymax = 0.1
+    zmin = 0.
+    zmax = 2.
+  []
+  [assign_coarse_id]
+    type = CoarseMeshExtraElementIDGenerator
+    input = fmg_id
+    coarse_mesh = coarse_mesh
+    extra_element_id_name = coarse_element_id
+  []
+[]
+
+[Executioner]
+  type = SweepUpdate
+
+  richardson_abs_tol = 1e-8
+  richardson_rel_tol = 1e-8
+  richardson_value = eigenvalue
+
+  richardson_max_its = 1000
+  inner_solve_type = GMRes
+  max_inner_its = 20
+
+  fixed_point_max_its = 1
```

Comment:
This is a parameter for the multiphysics coupling, it s not about neutronics only results

Reply:
Right. I discussed it with @miaoyinb we have several multiapp iterations because Richardson iterations re-runs multiapps all the time so that 1 fixed point iteration is enough.